### DISCUSSION TITLE
how to continue after re-labelling in ASReview

### DISCUSSION THREAD

**Discussion by sparksfly13** [asreview/asreview#1535] [Q&A] [answered]
Hi! I am using ASReview for a scoping review and I have screened about 900 articles so far. Our team decided to exclude a certain type of article, which was included before. After going through the History and re-labelling those articles - around 300 of them - I now don't know how to continue screening. Because the ASReview model would have given me other articles to screen if I had excluded this set of articles from the beginning. Looking back, I have already reached my stopping criterion (50 irrelevant articles in a row) after the re-labelling process.

I could of course continue screening until I reach the stopping criterion again. But I could also re-upload this project into ASReview with the prior knowledge of labelled articles. Perhaps with a different model, or the same one?

Can anyone give me advice on how to continue? I cannot imagine this is the first time someone has re-labelled articles while using ASReview.

**Comment by Rensvandeschoot** [comment, marked as answer]
Thank you for your detailed question. You're correct that ASReview takes into account all labeled data in each new iteration. If you've changed labels via the history panel, these updated labels will be used in the next iteration. For full reproducibility, these changes are stored in the `.asreview` file (see also [this paper](https://psyarxiv.com/g93zf/)).

Exporting and re-importing your project would essentially yield the same results. However, changing the model could be beneficial, as indicated in [this study](https://www.frontiersin.org/articles/10.3389/frma.2023.1178181/full) and [this process paper](https://psyarxiv.com/c93gq).

Given your situation, I would recommend conducting a quality check round, similar to what we did in [this paper](https://psyarxiv.com/j6nqz). Use the labels of the relevant papers from your current pipeline and label an additional 10-20 irrelevant papers. Train a new model with this data and then screen the most likely relevant papers that were excluded in the first round. This will help ensure that you haven't missed any relevant papers due to your changed inclusion criteria.
